### ISSUE TITLE
When the keyboard focus is on the "OK" button and the TAB key is pressed, the focus moves to the subscription list, but focus is not visible: A11y_VS Code_Appling filter on subscription_keyboard

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#245221]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG2.4.7; #FTP; #Focus Visible; #Keyboard;

## Environment Details:
Application: VS Code
Visual studio code version: 1.96.3
OS: Windows 11 Enterprise 24H2
Build: 26100.2605

## Repro Steps

1. Launch the VS Code.
2. Install the Azure Machine Learning extension and sign in with appropriate Credentials.
3. Tab until the left toolbar (list of icons). This toolbar has a file icon, search icon, settings, etc.
4. Use the up/down arrow keys to navigate to the "Azure" icon and press enter
5. The Azure panel will now be active next to the toolbar.
6. Tab until the Machine Learning dropdown.
7. Tab till any subscription for example AML V1 Personal subscription.
8. Tab until the "Filter" button and press Enter. It will be a button with a funnel icon.
9. TAB till "Select subscriptions" list and press down arrow key move focus to any list item.
10. Press Shift + TAB key to move "Ok" button.
11. Now press TAB key to move focus to list.
12. Observer that the focus moves to the subscription list, but focus is not visible.

## Actual Experience
When the keyboard focus is on the "OK" button and the TAB key is pressed, the focus moves to the subscription list, but focus is not visible and on next tab visible on first list item. 
Note: Make sure before repro'ing issue first move to list with arrow key.

## Expected Experience
When the keyboard focus is on the "OK" button and the TAB key is pressed, the focus should be visible.

## User Impact:
Keyboard only user will face difficulty if they press TAB key and keyboard focus is not visible.

## Attachment

![Image](https://github.com/user-attachments/assets/399fd483-efff-4d35-8655-3763f024690b)

**Comment by TylerLeonhardt** [comment]
Better steps:
* Open any multi-quickpick - like picking an account after running the `>Accounts: Manage Trusted Extensions For Account` command
* Hit down arrow a few times so that the highlighted item is not the first item
* Tab until "Cancel" is focused, then tab one more time
* 🐛 there is no indication that focus is on the list

**Comment by meganrogge** [comment]
@TylerLeonhardt as it is currently graded, we have to fix this by the end of May.

**Comment by TylerLeonhardt** [comment]
We should downgrade it

**Comment by Mana-swi** [comment]
#a11ydb-focus-missing-no-cue;
